kornia/geometry/conversions.py

```diff
@@ -534,21 +515,11 @@ def quaternion_to_rotation_matrix(
     if order.name not in QuaternionCoeffOrder.__members__.keys():
         raise ValueError(f"order must be one of {QuaternionCoeffOrder.__members__.keys()}")
 
-    if order == QuaternionCoeffOrder.XYZW:
-        warnings.warn(
-            "`XYZW` quaternion coefficient order is deprecated and"
-            " will be removed after > 0.6. "
-            "Please use `QuaternionCoeffOrder.WXYZ` instead."
-        )
-
     # normalize the input quaternion
     quaternion_norm: Tensor = normalize_quaternion(quaternion)
 
     # unpack the normalized quaternion components
-    if order == QuaternionCoeffOrder.XYZW:
-        x, y, z, w = torch.chunk(quaternion_norm, chunks=4, dim=-1)
-    else:
-        w, x, y, z = torch.chunk(quaternion_norm, chunks=4, dim=-1)
+    w, x, y, z = torch.chunk(quaternion_norm, chunks=4, dim=-1)
```

Comment:
```suggestion
    w = quaternion_norm[…, 0]
    x = quaternion_norm[…, 1]
    y = quaternion_norm[…, 2]
    z = quaternion_norm[…, 3]
```
to make this more pythonic. If you see more stuff like that feel free to fix